From d2813c02131b9ddf938277f4123da7ccbd113ea7 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 24 Jun 2024 22:35:42 +0100
Subject: [PATCH 1142/1145] fixup! drivers: mmc: sdhci-brcmstb: bcm2712
 supports HS400es and clock gating

Declaring auto-clockgate support for a host that can interface with
SDIO cards is a bug.

See: https://github.com/raspberrypi/linux/issues/6237

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 drivers/mmc/host/sdhci-brcmstb.c | 1 -
 1 file changed, 1 deletion(-)

--- a/drivers/mmc/host/sdhci-brcmstb.c
+++ b/drivers/mmc/host/sdhci-brcmstb.c
@@ -430,7 +430,6 @@ static const struct brcmstb_match_priv m
 };
 
 static const struct brcmstb_match_priv match_priv_2712 = {
-	.flags = BRCMSTB_MATCH_FLAGS_HAS_CLOCK_GATE,
 	.hs400es = sdhci_brcmstb_hs400es,
 	.cfginit = sdhci_brcmstb_cfginit_2712,
 	.ops = &sdhci_brcmstb_ops_2712,
